### PR TITLE
Chore dependency workflow

### PR DESCRIPTION
**Pull request by naseberry**
## What

Change the frequency of package update to be weekly,
And create Jira tickets to reflect the PR for visibility

## Checklist

Before you ask people to review this PR:

- Tests and linters should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

**Comment by naseberry** [comment]
For discussion, workflow changes based on the increasing number of repos maintained by the software team, and visibility for the project team.

**Comment by naseberry** [comment]
Note: may wish to Include and discuss @jsugarman  topic on grouping dependabots by package manager

File: .github/workflows/dependabot-jira.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    runs-on: ubuntu-latest
+    if: github.actor == "dependabot"
+
+    steps:
+      - name: Login
+        uses: atlassian/gajira-login@v3
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
```
Comment (naseberry) [inline .github/workflows/dependabot-jira.yml:19]: env values to be generated

**Review by jsugarman — APPROVED** [review]
weekly schedule looks good. jira ticket addition maybe should be separate PR but happy to try it out here, assuming env vars have been added.

**Comment by naseberry** [comment]
Re-opoend in https://github.com/ministryofjustice/laa-assure-hmrc-data/pull/1440

**Pull request closed without merging**